Mathlib/AlgebraicTopology/SimplexCategory/Basic.lean

```diff
@@ -411,11 +409,15 @@ lemma factor_δ_spec {m n : ℕ} (f : ⦋m⦌ ⟶ ⦋n+1⦌) (j : Fin (n+2))
   dsimp [factor_δ, δ, σ]
   cases j using cases with
   | zero =>
-    rw [predAbove_of_le_castSucc _ _ (zero_le _), castPred_zero, predAbove_of_castSucc_lt 0 _
-    (castSucc_zero ▸ pos_of_ne_zero hj),
-    zero_succAbove, succ_pred]
+    ext
+    unfold predAbove
+    simp only [castSucc_zero, lt_self_iff_false, ↓reduceDIte]
```

Comment:
Can the unfold be done by `simp`? Sorry for not checking myself, I have ropey internet right now.